Update troubleshooting guide for local disk costs. (#1048)
kubecost/docs: troubleshooting/troubleshoot-install.md

3. Change both entries to localhost:9001 and localhost:9003
4. Restart the kubecost-cost-analyzer pod in the kubecost namespace

> ### What is the difference between `kubecostToken` and `productKey`?

Author: Fixed! Removed question mark.

```
{% endcode %}

### Local disks showing costs in Assets

> Some cloud providers do not charge you for local disks physically attached to the node (i.e. ephemeral storage). By default, Kubecost monitors your local disk usage/capacity and applies an associated cost. To disable this feature use the following config:

Author: Ahh thank you! I always forget precisely what the it translates to in Latin. I previously always thought of "i.e." as in other words and "e.g." as for example ... which isn't _completely_ but also not completely accurate.

Yes you're right that in this instance "i.e. ephemeral storage" isn't technically accurate. I believe changing to "e.g." would correct this? Doing that now.

```yaml
kubecostModel:
  extraEnv:
    - name: "ASSET_INCLUDE_LOCAL_DISK_COST"

>       value: "false"

Author: Noted! Here's the PR I've started. https://github.com/kubecost/cost-analyzer-helm-chart/pull/3449

```

> This config will need to be applied to every cluster for which you do not want to charge for local disk cost. This config change only affects data moving forward. To fix historical data, you will need to [repair Asset & Allocation ETL](/troubleshooting/etl-repair.md) for each affected cluster, then wait for Kubecost's Aggregator to reingest the updated ETL data.

Reviewer: ```suggestion
This configuration will need to be applied to any cluster on which you do not want to charge for local disks and only takes effect on data moving forward. To fix historical data, you will need to [repair Asset & Allocation ETL](/troubleshooting/etl-repair.md) for each affected cluster, then wait for Kubecost's Aggregator to reingest the updated ETL data.
```
